### DISCUSSION TITLE
Custom body measurements

### DISCUSSION THREAD

**Discussion by astashov** [astashov/liftosaur#133] [Ideas]
It probably makes sense to allow people to create custom measurements.

From a Github issue:

I would like to be able to track additional metrics like

Metabolic markers
VO2 max
Resting energy
Walking heart rate average
Recovery indicators
Resting heart rate
Resting respiratory rate
Sleep
Heart rate variability
Cardio recovery
Subjective scores (e.g. tiredness/soreness)
Vital signs
Blood pressure
Body temperature
Activity stats
Steps
Distance walked / run
Stair flights climbed
These are just a subset of many possible measurements, so it might be worth allowing custom measurements so users aren't limited to a pre-defined list.

I know this is primarily a lifting app but as long as you aren't trying to sync these stats from somewhere else like Apple Health, I'd expect it to be a simple expansion of existing functionality. If you think this request is a good idea, I'd like to try writing a PR myself.

**Comment by s-boardman** [comment]
Muscle % would be ace.
